ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,577 @@
+                    primaryLabel.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -Constants.verticalCardPaddingBottom)
+                ])
+            }
+        case .announcement:
+            // featured image + title + subtitle + button
+            height += (style.imageHeight + Constants.verticalContentSpacing + Constants.verticalCardPaddingBottom)
+
+            layoutConstraints.append(contentsOf: [
+                heightAnchor.constraint(greaterThanOrEqualToConstant: height),
+                widthAnchor.constraint(greaterThanOrEqualToConstant: style.cardMinWidth),
+                primaryLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: Constants.largeHorizontalPadding),
+                primaryLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -Constants.largeHorizontalPadding),
+                secondaryLabel.topAnchor.constraint(equalTo: primaryLabel.bottomAnchor, constant: Constants.largeContentSpacing),
+                secondaryLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: Constants.largeHorizontalPadding),
+                secondaryLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -Constants.largeHorizontalPadding)
+            ])
+
+            if let button = button {
+                layoutConstraints.append(contentsOf: [
+                    button.centerXAnchor.constraint(equalTo: centerXAnchor),
+                    button.topAnchor.constraint(equalTo: secondaryLabel.bottomAnchor, constant: Constants.largeVerticalPadding),
+                    button.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -Constants.largeVerticalPadding)
+                ])
+            }
+
+            if let imageView = imageView {
+                layoutConstraints.append(contentsOf: [
+                    imageView.leadingAnchor.constraint(equalTo: leadingAnchor),
+                    imageView.trailingAnchor.constraint(equalTo: trailingAnchor),
+                    imageView.topAnchor.constraint(equalTo: topAnchor),
+                    imageView.heightAnchor.constraint(greaterThanOrEqualToConstant: style.imageHeight),
+                    primaryLabel.topAnchor.constraint(equalTo: imageView.bottomAnchor, constant: Constants.largeVerticalPadding)
+                ])
+            }
+        }
+
+        NSLayoutConstraint.activate(layoutConstraints)
+    }
+
+    open override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        if let previousTraitCollection = previousTraitCollection {
+            if previousTraitCollection.preferredContentSizeCategory != traitCollection.preferredContentSizeCategory {
+                setupLayoutConstraints()
+            }
+            if #available(iOS 13, *) {
+             if previousTraitCollection.hasDifferentColorAppearance(comparedTo: traitCollection) {
+                    setupColors()
```

Comment:
Most colors don't need to be updated in here. Only the ones converted to CGColor.